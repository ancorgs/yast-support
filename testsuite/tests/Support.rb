# encoding: utf-8

module Yast
  class SupportClient < Client
    def main
      # testedfiles: Support.ycp

      Yast.include self, "testsuite.rb"
      TESTSUITE_INIT([], nil)

      Yast.import "Support"

      DUMP("Support::Modified")
      TEST(lambda { Support.Modified }, [], nil)

      nil
    end
  end
end

Yast::SupportClient.new.main
