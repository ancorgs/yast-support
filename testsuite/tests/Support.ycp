{

// testedfiles: Support.ycp

include "testsuite.ycp";
TESTSUITE_INIT([], nil);

import "Support";

DUMP("Support::Modified");
TEST(``(Support::Modified()), [], nil);

}
