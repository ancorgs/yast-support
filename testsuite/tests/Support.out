Dump	Support::Modified
Return	false
